mechafil/data_spacescope.py

```diff
@@ -0,0 +1,280 @@
+import requests
+import pandas as pd
+import numpy as np
+import datetime
+from typing import Tuple, List
+
+EXBI = 2**60
+PIB = 2**50
+
+NETWORK_START = datetime.datetime(2020, 10, 15)
+DEFAULT_SPACESCOPE_CHUNK_SIZE_IN_DAYS = 90
+SPACESCOPE_AUTH_KEY = 'Bearer ghp_xJtTSVcNRJINLWMmfDangcIFCjqPUNZenoVe'
```

Comment:
- I agree, and I will change this so that there is a config file. 
- However, the key that I posted is the default key that is being shared by spacescope in its documentation. This isn't private to us or anybody at the moment, so we won't need to request a new one.

Reply:
I think a clean way of implementing user configuration is to convert the data.py to a Python class, with a required input of the bearer-token. That way, the token is associated w/ the query in the case where the backend requires the token.  Is that path forward OK w/ you, or did you want to keep things non-OOP?  I think it would be more kludgey to have non-oop based method of getting user input b/c you'd either
a) store the user info in a global variable
b) pass user config info into every function call

OOP allows that user info to be associated w/ the object so it doesn't need to be passed around every time.

Reply:
The solution sounds good to me. I agree that using a class will be cleaner.

Reply:
> * I agree, and I will change this so that there is a config file.
> * However, the key that I posted is the default key that is being shared by spacescope in its documentation. This isn't private to us or anybody at the moment, so we won't need to request a new one.

I was not aware of this. In that case, no need to ask spacescope to delete it.